src/components/Firestore/Requests/RequestDetails/ActionsCollapsedMenu.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import './ActionsCollapsedMenu.scss';
+
+import { IconButton } from '@rmwc/icon-button';
+import { MenuItem, SimpleMenu } from '@rmwc/menu';
+import React, { useState } from 'react';
+
+import { RequestAction } from './Actions';
+
+interface Props {
+  requestActions: RequestAction[];
+}
+
+const RequestActionsCollapsedMenu: React.FC<Props> = ({ requestActions }) => {
```

Comment:
nit: This component can probably just be part of `Actions.tsx`. That should also make CSS importing much easier.